### PR TITLE
Fdc3 explained 1.0.1

### PR DESCRIPTION
**Pull request by WatsonCIQ**
Updates: 
- Added raise intent
- Updated Context Listening
- Updated Broadcast
- Temp removed app loader spawning
- Small visual tweaks for the table
- Updated Joining Channels
- Ability to add  FDC3 app channels 
- Add the FDC3 Ready pattern to check for FCDC3 Supported

### CODE REVIEW

File: toolbox/fdc3-explained/index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<html>
+
+<head>
+	<title>
+		FDC3 Explained v.2
+	</title>
+
+
+
```
Comment (kriswest) [inline toolbox/fdc3-explained/index.html]: When run locally was picking up the wrong character encoding:
image
Explicit encoding set resolves that
```suggestion
<meta charset="UTF-8">
```

File: toolbox/fdc3-explained/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+			<tr>
+				<td colspan="3">
+					<hr>
+				</td>
+			</tr>
+
+			<tr>
+				<td class="header">App Directory:</td>
+				<td><span id="directoryDetails">No Application Directory Connected</span></td>
```
Comment (kriswest) [inline toolbox/fdc3-explained/index.html]:
```suggestion
				<td><span id="directoryDetails">Not supported yet!</span></td>
```

File: toolbox/fdc3-explained/index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+					<hr>
+				</td>
+			</tr>
+
+			<tr>
+				<td class="header" rowspan="3">Raise Intent:</td>
+			</tr>
+
+			<tr>
+				<td>Intent: <input type="text" id="intent"></td>
```
Comment (kriswest) [inline toolbox/fdc3-explained/index.html]:
```suggestion
				<td>Intent: <input type="text" id="intent" value="ViewChart"></td>
```

File: toolbox/fdc3-explained/index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+			</tr>
+
+			<tr>
+				<td><textarea id="intent-context"></textarea></td>
```
Comment (kriswest) [inline toolbox/fdc3-explained/index.html]:
```suggestion
				<td><textarea id="intent-context">{"id":{"ISIN":"US0378331005","SEDOL":"2046251","ticker":"AAPL"},"name":"Apple Inc.","type":"fdc3.instrument"}</textarea></td>
```

**Review by kriswest — APPROVED** [review]
Works for me in both Nick's extension and Finsemble and retains the easy to follow low-level javascript approach. Splitting the file into CSS/HTML/JS aids legibility further. 

I added a few suggestions - it needed an explicit charset, I think it worth noting that the app directory functionality is not wired up yet and adding defaults for raiseIntent (otherwise we should add error handling code).

The next step should probably be to add AppD retrieval and raiseIntentForContext support. Channel API options (broadcast and subscribe) if desired.

Approving - @WatsonCIQ if you agree with suggestions, commit 'em.

**Comment by donbasuno** [comment]
Looks good to me!

**Pull request merged**